# Задание 1

Вы измеряете физическую величину, например, давление, следите, чтобы оно не превышало 5 атм. 
Получили ряд измерений, при этом условия процесса не менялись. Давление на самом деле было постоянным.
Измерения: *5.2*, *4.3*, *4.7*, *4.9*, *5.5* атм.

С какой вероятностью давление:
  - **ниже 5 атм?**
  - **ниже 4 атм?**

---

## Решение

Поскольку ни сам метод измерения, ни погрешность измерительного прибора неизвестна, то оценим вероятность наступления двух событий:
 - событие A давление ниже 5 атм,
 - событие B давление ниже 4 атм,
 
на основе имеющегося ряда измерений, для этого применим классическое определение вероятности.

In [1]:
measurements = (5.2, 4.3, 4.7, 4.9, 5.5)

In [2]:
# Событие A
# есть отношение числа измерений, которые ниже 5 атм к общему количеству измерений
condition = 5.0
p_A = len([m for m in measurements if m < condition]) / len(measurements)

print(f'Вероятность того что давление ниже {condition} атм, равна = {p_A:0.3f}')

Вероятность того что давление ниже 5.0 атм, равна = 0.600


In [3]:
# Событие B
# есть отношение числа измерений, которые ниже 4 атм к общему количеству измерений
condition = 4.0
p_B = len([m for m in measurements if m < condition]) / len(measurements)

print(f'Вероятность того что давление ниже {condition} атм, равна = {p_B:0.3f}')

Вероятность того что давление ниже 4.0 атм, равна = 0.000


## Ответ

Вероятность того, что давление ниже 5.0 атм, равна = 0.60

Вероятность того, что давление ниже 4.0 атм, равна = 0.00